src/main/webapp/gameStage.html

```diff
@@ -13,8 +13,15 @@
   <script src="gameDialogue.js"></script>
 </head>
 <body onload = "loadFunctions()">
+    
+  <audio controls autoplay loop style="float:left;"> <!--hidden tag can be added-->
```

Comment:
I like the idea of it being hidden, but with a mute button so its really an on/off switch?